app/js/lib/common/event-dispatcher.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
```

Comment:
This file comes with unit test. We can defer adding them to later but in the meantime, can you open an issue so that we don't forget about it?